docs/reference/release-notes/highlights-6.3.0.asciidoc

```diff
@@ -4,6 +4,45 @@
 <titleabbrev>6.3.0</titleabbrev>
 ++++
 
-coming[6.3.0]
+Here are the highlights of features that were added in Elasticsaerch 6.3.0.
 
-See also <<release-notes-6.3.0,{es} 6.3.0 release notes>>. 
+Refer to the <<release-notes-6.3.0,{es} 6.3.0 release notes>> for the full list
+of changes in this release.
+
+NOTE: We want your feedback on the experiemental features in this release! Let
+us know what you’d like to see next and any problems you encounter.
+
+=== SQL
+This experimental feature enables users who are familiar with SQL to
+use SQL statements to query Elasticsearch indices. In addition to querying
+through the SQL API, you can use the Translate API to see how SQL queries
+are translated to native ELasticsearch queries.
+
+The included SQL CLI provides a simple way to submit SQL queries to Elasticsearch.
+Similarly, the Elasticsearch JDBC driver enables you to connect your favorite
+JDBC-compatible tool to Elasticsearch.
+
+For more information, see <<xpack-sql, SQL Access>>.
+
+=== Rollups
+This experimental feature enables you to summarize and store historical data
+so that is still available for analysis, but consumes significantly less
+storage space. This is particularly useful when you're working with monitoring
+and metrics data where it's not feasible to retain the raw data indefinitely.
+
+When you ask Elasticsearch to store a rollup of data, it also stores the underlying
+statistics. For example, if you roll up an average, the sum and count are also
+stored so that the average can be recomputed at query time. This enables you to
+query both rolled up data and “live” data simultaneously using the standard
+query DSL.
+
+For more information, see <<xpack-sql, SQL Access>>.
```

Comment:
shouldn't this be a link to rollups instead of SQL?

Reply:
Yup. Darned copy/paste. :-)